sdk/core-v2/azure-core/src/main/java/com/azure/v2/core/http/polling/Poller.java

```diff
@@ -159,4 +165,80 @@ default Poller<T, U> setPollInterval(Duration pollInterval) {
         return this;
     }
 
+    /**
+     * Creates default SyncPoller.
```

Comment:
nit; Not needed right now but we should rename these to drop the 'sync' prefix from variables and docs.
```suggestion
     * Creates default Poller.
```